calico/common.py

```diff
@@ -71,18 +75,27 @@
     "protocol",
     "src_net",
     "src_tag",
+    "src_selector",
     "src_ports",
     "dst_net",
     "dst_tag",
+    "dst_selector",
     "dst_ports",
     "icmp_type",
     "icmp_code",
     "ip_version",
 ])
 
+# Valid actions to see in a rule.
+KNOWN_ACTIONS = set(["allow", "deny", "next-tier"])
```

Comment:
Could we use the { } syntax, now that we're not supporting Python 2.6?

I guess the answer is that there isn't so much benefit from the { } syntax that it's worth creating an extra obstacle just in case we had to support Python 2.6 again.  But just raising this anyway in case there are stronger reasons why it might be good now to switch to { }.